packages/mobile-top-nav/src/media-menu.js

```diff
@@ -9,52 +9,88 @@ class MediaMenu extends LitElement {
       mediaMenuOpen: { type: Boolean },
       mediaMenuAnimate: { type: Boolean },
       mediaSliderOpen: { type: Boolean },
-      mediaSliderAnimate: { type: Boolean }
+      mediaSliderAnimate: { type: Boolean },
+      tabHeightMultiplier: { type: Number },
     };
   }
 
   constructor() {
     super();
     this.mediaSliderOpen = false;
     this.mediaSliderAnimate = false;
+    this.tabHeightMultiplier = 0;
   }
 
-  mediaSlider() {
+  updated(changedProperties) {
+    const { mediaMenuOpen, mediaSliderOpen } = this;
+    const menuClosed =
+      changedProperties.has('mediaMenuOpen') &&
+      changedProperties.get('mediaMenuOpen') &&
+      !mediaMenuOpen;
+
+    if (menuClosed && mediaSliderOpen) {
+      this.mediaSliderOpen = false;
+      this.mediaSliderAnimate = false;
+    }
+  }
+
+  mediaSlider(tabMultiplier) {
     // Only keep side menu open until menu closed
     // Shift menu downwards as per button clicked
-    this.mediaSliderAnimate = true;
-    this.mediaSliderOpen = !this.mediaSliderOpen;
+    if (!this.mediaSliderOpen) {
+      this.mediaSliderAnimate = true;
+      this.mediaSliderOpen = !this.mediaSliderOpen;
+    }
+    this.tabHeightMultiplier = tabMultiplier;
   }
 
   render() {
-    const mediaMenuClass = this.mediaMenuOpen ? 'media-menu open slide-in' : this.mediaMenuAnimate ? 'media-menu slide-out' : 'media-menu';
+    let mediaMenuClass = '';
+    if (this.mediaMenuOpen) {
```

Comment:
yes, agree it can be abstracted out.  this isn't the last time this file will be touched, will convert then. main premise will be paring down DOM and CSS